recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+channel_targets:
+  - conda-forge bw2io_dev
+channel_sources:
+  - conda-forge/label/bw2data_dev,conda-forge,defaults
```

Comment:
1. Why we need a pre-release of `bw2data` here?
2. `defaults` should not be added here. We should stay inside `conda-forge`
3. Please always add an empty line add the end

Reply:
ad 1.: I am not sure we technically need to add `bw2data` to the `channel_sources`, since the current requirements (of the `dev` version of `bw2io` specify only [`"bw2data>=3.5.1"`](https://github.com/brightway-lca/brightway2-io/blob/0c3c7288a897f57511ce17a6be1698e2cb9b08a1/setup.py#L5C5-L5C22). I added this while following the [CFEP-05 guide here](https://github.com/conda-forge/cfep/blob/main/cfep-05.md#specification), but I'll remove it for now.
ad 2.: ✅
ad 3.: ✅